build.gradle

```diff
@@ -37,7 +37,7 @@ javadoc {
 dependencies {
     compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
 
-    implementation 'org.eclipse.jgit:org.eclipse.jgit:6.4.0.202211300538-r'
+    implementation 'org.eclipse.jgit:org.eclipse.jgit:6.5.0.202303070854-r'
```

Comment:
This should be in a different commit with a dedicated ticket about the upgrade, so that it's easy to track it down if we end up having side effects due to this upgrade.

Reply:
I will revert that change. 
In SC this was overridden  by version 6.3.X and the performances were not good
We saw that in SQ, you are using 6.5.0.202303070854-r, so we did the same in SC and then blame was much faster
I will anyway revert it and create a new PR with the upgrade

Reply:
Done